cmd/thanos/bucket.go

```diff
@@ -375,6 +379,38 @@ func registerBucketWeb(m map[string]setupFunc, root *kingpin.CmdClause, name str
 	}
 }
 
+func registerBucketReplicate(m map[string]setupFunc, root *kingpin.CmdClause, name string, objStoreConfig *extflag.PathOrContent) {
+	cmd := root.Command("replicate", "Replicate data from one object storage to another. NOTE: Currently it works only with Thanos blocks (meta.json has to have Thanos metadata).")
```

Comment:
Minor nit: we can use `block.MetaFilename` here.

Reply:
Sorry, do not get your meanning :(

Reply:
That's the constant that has the value "meta.json" so we could do fmt.Sprintf here :P

Reply:
Get it, thanks for your patience.